### ISSUE TITLE
[Feature] Allow team to switch between time tracking AND working days / due date tracking for tasks

### ISSUE THREAD

**Issue by evereq** [ever-co/ever-teams#1396]
2 Modes for Ever Teams operation:

- Time tracking for issues (Current mode)
- Use Due dates / Start Dates for tracking issues' progress. In this mode, we just hide the clock and just display the "Start / Stop" button that members can use to show if s/he working / not working. We also replace "progress" and "time estimate" with progress that uses Start Date / Due Dates and instead of time estimate, we allow to enter those Start Date / Due Date values for a given task (on member/task card)

Of course, that should be a setting for the Team / Project level, not individual members.

**Comment by KostyaEver** [comment]
Maybe we can have actually selector for 3 options: a) use time tracking and estimates OR b) don't use time tracking and use Start / Due dates for tasks OR c) hybrid - some tasks can use estimates, others due dates.
If it's Hybrid mode, maybe we should Add ability to select not only estimate, but also start date / due date in that place of main input (e.g. put those 2 inputs below Estimate input for example?). So that when user create (or edit) task s/he can decide to enter Due date instead (or in addition to Estimate).
Same as 4, depending on "Mode" (tracking/not-tracking/hybrid) we will decide what to do on cards (it's tricky, require you to think and write it down and I will review)


(each Team select mode in Settings, I think default can be "hybrid")

Think about
- Hybrid - Time tracking & No Tracker
- Issue type - if we can hide
- Main input - modify under new request
- Which columns we leave / add (Start date / Due date) may be marker Column - 'Days left' etc
- Tasks page (Worked/Assigned/Unassigned) should be adapted
- Task Details should be adapted

**Comment by evereq** [comment]
@JoeMakuta can you design that in Figma? I think it's a good feature to implement. It must be a setting for each Team (selection mentioned in above comment). And next, you need to work on multiple designs for team / task page accordingly.

**Comment by KostyaEver** [comment]
Also I would do next instead the 'Total Worked today' I would have 'Days Left' (so we know how many days until delivery)

<img width="816" alt="Screenshot at May 30 08-21-06" src="https://github.com/ever-co/ever-teams/assets/107203576/4323e6fa-7638-49cd-8f56-4c7d2c6b219c">

@evereq @JoeMakuta

**Comment by JoeMakuta** [comment]
### Here is my description of how I think the feature can work.
3 Modes for Ever Teams : Time tracking, Start / due dates and Hybrid  (Each team select the mode in Team settings, Default : Hybrid)

1. Time tracking mode (Current mode  in v.2)
 
2. Start / due dates mode
      - The user sets start and due dates for a task (not time estimations)
      - We hide the clock and just display the "Start / Stop" button to show if s/he working / not working
      - On task card, we display number of remaining days before due date -> "Days left"

3. Hybrid mode (The combined of 1st and 2nd modes)
      - We display everything from Time tracking and Start / Due dates modes.
      
      
@evereq @KostyaEver, Can you suggest something ?

**Comment by evereq** [comment]
Yes @JoeMakuta , now you need to show designs for all of that in Figma, e.g. like @KostyaEver suggested some cards will show one thing, others another etc, depending on tracking method, etc.
